root/frontend/src/components/PreviewCard/PreviewCard.style.ts

```diff
@@ -0,0 +1,68 @@
+import { createUseStyles } from 'react-jss';
+
+const useStyles = createUseStyles((theme: any) => ({
+  previewCard: {
+    width: '200px',
+    background: '#4E525B',
+    maxHeight: '250px',
+    borderRadius: '15px',
+    color: theme.color,
```

Comment:
I think we should probably update all the `theme.color` to take in a specific given colour since each card will have independent colours; likely just add a prop and pass that through the styles. If you need help with how to do that just let me know, or I can make a commit to this branch to add it.

Reply:
`theme.color` is dynamic from the `theme` variable in `root/frontend/src/components/PreviewCard/PreviewCard.tsx`